recipes-multimedia/cenc/gstreamer1.0-plugins-cencdecrypt_git.bb

```diff
@@ -16,7 +17,7 @@ DEPENDS = " \
 PV = "0.1.gitr${SRCPV}"
 
 SRC_URI = "git://github.com/WebPlatformForEmbedded/gstcencdecryptor.git;protocol=https;branch=master"
```

Comment:
Can you please add BRANCH variable and make weak assignment for BRANCH and SRCREV?

Reply:
done